config/config.go

```diff
@@ -209,7 +216,7 @@ func (rc *RepoConfig) parseDocument(repoName repos.RepoName, doc jsonDoc) error
 			return err
```

Comment:
Strange formatting, could you run gofmt in separate PR?

Reply:
This is already formatted with gofmt, otherwise the build would have failed. See: https://github.com/daedaleanai/reqtraq/runs/6721802746?check_suite_focus=true